README.md

```diff
@@ -1,18 +1,15 @@
-Windows Azure Active Directory IdentityModel Extensions for .Net
-===========
+# Windows Azure Active Directory IdentityModel Extensions for .Net
+
+[![Nuget](https://img.shields.io/nuget/v/Microsoft.IdentityModel.JsonWebTokens?label=Latest%20release)](https://www.nuget.org/packages/Microsoft.IdentityModel.JsonWebTokens/)
 
 IdentityModel Extensions for .NET provide assemblies that are interesting for web developers that wish to use federated identity providers for establishing the caller's identity.
 
 ## Versions
-Current version - 6.6.0 \
-Minimum recommended version - 5.5.0
 
 You can find the release notes for each version [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases). Older versions can be found [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/wiki/Release-Notes).
 
-## 6.6.0
-See the release notes [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/6.6.0)
-
 ## Note about 6.x
+
```

Comment:
Small changes like these help the readability when not rendered. I made a few whitespace changes based on current best practices for markdown.